CMakeLists.txt

```diff
@@ -54,14 +54,15 @@ enable_language(CXX)
 
 
 list(INSERT CMAKE_MODULE_PATH 0 ${PROJECT_SOURCE_DIR}/cmake)
-include(cc_rules)
 include(compiler_id)
 include(external_rules)
 include(podspec_rules)
 include(archive_options)
 include(sanitizer_options)
 include(fuzzing_options)
 
+include(cc_rules)
```

Comment:
Perhaps add a comment to note that `cc_rules` has to be included after `compiler_id`?

Reply:
Done. And for consistency moved all the rules down here, clearly separating options processing from rule definition.